frontend/api/errors.py

```diff
@@ -13,6 +13,8 @@
 # modified, propagated, or distributed except according to the
 # terms contained in the LICENSE file.
 
+import sys
+import os
```

Comment:
You're right, i don't need os, but still need sys to display the current exception caught by the API. I didnt succeeded in making bottle logging through uwsgi so directly print it for the moment.